app/assets/javascripts/discourse/app/components/user-menu/menu.hbs

```diff
@@ -0,0 +1,50 @@
+<div class="user-menu revamped menu-panel drop-down" data-max-width="320">
+  <div class="panel-body">
+    <div class="panel-body-contents">
+      <div
+        id={{concat "quick-access-" this.currentTabId}}
+        class="quick-access-panel"
+        tabindex="-1"
+        aria-labelledby={{concat "user-menu-button-" this.currentTabId}}>
+        <this.currentPanelComponent/>
+      </div>
+      <div class="menu-tabs-container" role="tablist" aria-label={{i18n "user_menu.sr_menu_tabs"}}>
+        <div class="top-tabs tabs-list">
+          {{#each this.topTabs as |tab|}}
+            <button
+              class={{if (eq tab.id this.currentTabId) "btn btn-flat btn-icon no-text active" "btn btn-flat btn-icon no-text"}}
+              type="button"
+              role="tab"
+              id={{concat "user-menu-button-" tab.id}}
+              tabindex={{if (eq tab.id this.currentTabId) "0" "-1"}}
+              aria-selected={{if (eq tab.id this.currentTabId) "true" "false"}}
+              aria-controls={{concat "quick-access-" tab.id}}
+              data-tab-number={{tab.position}}
+              {{on "click" (fn this.changeTab tab)}}
+              {{!-- template-lint-disable require-context-role --}}
```

Comment:
It looks like this role needs adjusting because it's complaining about this element not being inside an element with `role="tablist"`, when it actually is but it's not an immediate child. As per [the spec](https://www.w3.org/TR/wai-aria-1.1/#tab), it doesn't have to be an immediate child.